### ISSUE TITLE
Auto-capitalize various entities

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/monarch-legacy#267]
I noticed in the match viewer, we see both capitalized and uncapitalized phenotypes from uberpheno. Perhaps these can be all capitalized for consistency throughout the site. Genes and genotypes should never be changed in their capitalization, but other entities could for consistency (diseases, later anatomy).

**Comment by cmungall** [comment]
Just in the match viewer, or universally throughout the system?

The cleanest place to do this is upstream. Why capitalized? I think this should be done if we are translating structured data into narrative sentences, terms at the start of a sentence should be capitalized.

Seems not uncommon to follow the normal rules of the english language and only capitalized at the start of the sentence, even in clinical settings. See:
- http://www.nlm.nih.gov/medlineplus/ency/article/003271.htm
- https://en.wikipedia.org/wiki/Short_stature

TL;DR - HPO is broken it needs to decapitalize except proper names :-)

AT LEAST ITS BETTER THAN OMIM

**Comment by jmcmurry** [comment]
@cmungall, should this instead be an HPO ticket then?

**Comment by cmungall** [comment]
No, HPO will not change their style

**Comment by jmcmurry** [comment]
Well, agreed but if they were to somehow (hand) flag the terms that are indeed proper names, we could capitalize or decapitalize deterministically. I'm guessing it is a low priority for them and for us, but please chime in.

**Comment by jmcmurry** [comment]
@cmungall capital offense territory?
